src/explore-education-statistics-admin/src/pages/release/datablocks/components/chart/ChartDataSetsConfiguration.tsx

```diff
@@ -197,62 +201,108 @@ const ChartDataSetsConfiguration = ({
 
       {dataSets?.length > 0 && (
         <>
-          <table>
-            <thead>
-              <tr>
-                <th>Data set</th>
-                <th>
-                  <VisuallyHidden>Actions</VisuallyHidden>
-                </th>
-              </tr>
-            </thead>
-            <tbody>
-              {dataSets.map((dataSet, index) => {
-                const expandedDataSet = expandDataSet(dataSet, meta);
-                const label = generateDataSetLabel(expandedDataSet);
-
-                return (
-                  <tr key={generateDataSetKey(dataSet)}>
-                    <td>{label}</td>
-                    <td className="dfe-align--right">
-                      <ButtonText
-                        className="govuk-!-margin-bottom-0"
-                        onClick={() => {
-                          if (onChange) {
-                            const nextDataSets = [...dataSets];
-                            nextDataSets.splice(index, 1);
+          <DragDropContext
+            onDragEnd={result => {
+              if (!result.destination) {
+                return;
+              }
+              const reorderedDataSets = reorder(
+                dataSets,
+                result.source.index,
+                result.destination.index,
+              );
+              onChange(reorderedDataSets);
+            }}
+          >
+            <Droppable droppableId="droppable" isDropDisabled={!isReordering}>
+              {(droppableProvided, droppableSnapshot) => (
+                <ol
+                  ref={droppableProvided.innerRef}
+                  className={classNames('govuk-list', {
+                    [styles.dropArea]: droppableSnapshot.isDraggingOver,
+                  })}
+                >
```

Comment:
Is it a good idea to swap to using a list? We've done table ordering before (see `LegacyReleasesTable`)

We also might want to add other columns to this in the future so a table would probably be more suitable?

Reply:
I thought simplifying the markup to an ordered list was more suitable as it's not tabular data, it's just a list of labels and remove buttons. It can always be changed to a table later if more data is added, but I'm not sure what that would be.

Reply:
I guess it depends on what the definition of tabular data is, but I'd argue this is tabulated :laughing:

Whilst adding new columns isn't clear now, it'd be a wasted effort to have to revert this back to a table later so I'd err on the side of caution and future proof it anyway.

From an accessibility standpoint, changing this to a list is arguably worse as buttons are displayed on every item and might be harder to skip when you're not interested in them. With a table, we were previously telling users (with visually hidden text) that the right column is for actions, meaning they know to skip that column if they're not interested in it whilst traversing the table rows.

We're also displaying tables with a similar structure using action columns in other places of admin too. It'd be good just from a consistency standpoint to keep the patterns the same as well.